packages/storage/amplify_storage_s3/example/integration_test/use_case_test.dart

```diff
@@ -696,52 +657,6 @@ void main() {
             );
           });
 
-          testWidgets(
-              skip: true,
-              'move object with access level guest for the currently signed in user',
-              (WidgetTester tester) async {
-            final result = await Amplify.Storage.move(
-              source: S3ItemWithAccessLevel(
-                storageItem: S3Item(key: testObjectKey1),
-                accessLevel: StorageAccessLevel.guest,
-              ),
-              destination: S3ItemWithAccessLevel(
-                storageItem: S3Item(key: testObject1MoveKey),
-                accessLevel: StorageAccessLevel.private,
-              ),
-              options: const StorageMoveOptions(
-                pluginOptions: S3MovePluginOptions(
-                  getProperties: true,
-                ),
-              ),
-            ).result;
-
-            expect(result.movedItem.eTag, isNotEmpty);
-          });
-
-          testWidgets(
-              skip: true,
-              'copy object (belongs to other user) with access level protected'
-              ' for the currently signed in user', (WidgetTester tester) async {
-            final result = await Amplify.Storage.copy(
-              source: S3ItemWithAccessLevel.forIdentity(
-                user1IdentityId,
-                storageItem: S3Item(key: testObjectKey2),
-              ),
-              destination: S3ItemWithAccessLevel(
-                storageItem: S3Item(key: testObject2MoveKey),
-                accessLevel: StorageAccessLevel.private,
-              ),
-              options: const StorageCopyOptions(
-                pluginOptions: S3CopyPluginOptions(
-                  getProperties: true,
-                ),
-              ),
-            ).result;
-
-            expect(result.copiedItem.eTag, isNotEmpty);
-          });
```

Comment:
Why is this test being removed?

Reply:
I think I removed this because it depended on a test I was removing and it seemed redundant to other e2e tests we have. I added it back though and updated it to no longer be dependent on the other test. We can decide if a test like this is needed or not when we refactor the full test suite.